### ISSUE TITLE
[Occtax] - Gestion des champs horaires

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/GeoNature#1557]
Actuellement, la gestion des horaires dans occtax pose question, sujet notamment abordé dans le cadre du mobile ici : https://github.com/PnX-SI/gn_mobile_occtax/issues/104

En web :
- soit on ne renseigne pas d'heure : le champs date contient alors JJ/MM/AAAA 00:00:00 et les champs hour_min hour_max sont NULL. 
- Soit on saisit une heure, on a alors JJ/MM/AAAA 12:19:03 et les champs hour sont renseignés avec 12:19:03

Le plus pertinent semblerait que les champs dates soient au format date et non pas timestamp, afin que l'information de l'heure soit uniquement renseignée dans les champs heure.

**Comment by bouttier** [comment]
Je suis d’accord, le champs date de type timestamp me semble être une erreur.

La synthèse pourrait également évoluer pour un champs date + un champs heure comme le standard (qui a évolué en ce sens). Cela éviterait également dans le module d’import des concaténations de date + heure pour alimenter la synthèse.

**Comment by DonovanMaillard** [comment]
oui, idem pour les sous-modules monitorings etc, et j'imagine que ca peut aussi simplifier les requêtes sur les heures etc.
